Sprint_7/src/test/java/CourierTest.java

```diff
@@ -0,0 +1,58 @@
+
+import io.restassured.response.ValidatableResponse;
+import org.example.Courier;
+import org.example.CourierClient;
+import org.example.CourierCredentials;
+import org.example.CourierGenerator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.apache.http.HttpStatus.*;
+import static org.junit.Assert.assertEquals;
+
+public class CourierTest {
+    private Courier courier;
+    private CourierClient courierClient;
+    private int id;
+
+    @Before
+    public void setUp(){
+        courier = CourierGenerator.getDefault();
+        courierClient = new CourierClient();
+    }
+
+    @After
+    public void cleanUp(){
+        if (id != 0 ){
+            courierClient.delete(id);
+        }
+    }
+
+    @Test
+    public void courierCanBeCreated(){ //создание курьера и авторизация
+        ValidatableResponse response = courierClient.create(courier);//создай курьера
+        ValidatableResponse loginResponse = courierClient.login(CourierCredentials.from(courier));//авторизация
+        id = loginResponse.extract().path("id");//тащит id
```

Comment:
Здесь возникает ошибка при запуске теста:
java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.response.ExtractableResponse.path(String, String[])" is null